PKGBUILD

```diff
@@ -0,0 +1,76 @@
+# Maintainer: hodasemi <michaelh.95 at t-online dot de>
+_pkgbase=breezy-desktop
+pkgname="${_pkgbase}"
+pkgver=0.1
+pkgrel=1
+pkgdesc="Breezy desktop - XR desktop"
+arch=('x86_64')
+url="https://github.com/wheaney/breezy-desktop"
+license=('GPL-3.0')
+makedepends=('ninja' 'meson' 'librsvg')
+depends=('python' 'python-pydbus' 'gnome-shell' 'XRLinuxDriver')
+conflicts=("${_pkgbase}")
+source=("git+${url}")
+md5sums=(SKIP)
+
+_uuid="breezydesktop@xronlinux.com"
+
+build() {
+    cd ${_pkgbase}
+
+    # init submodules (only required ones)
+    git submodule update --init --recursive modules/sombrero
+    git submodule update --init --recursive ui/modules/PyXRLinuxDriverIPC
+
+    # build binaries
+    cd ui
+    meson setup build
+    cd build
+    meson compile
+
+    # prepare extension
+    cd ../..
+    unlink gnome/src/schemas/com.xronlinux.BreezyDesktop.gschema.xml
+    cp ui/data/com.xronlinux.BreezyDesktop.gschema.xml gnome/src/schemas/
+    glib-compile-schemas --targetdir="gnome/src/schemas" "gnome/src/schemas"
```

Comment:
So the UI app is not actually running as a flatpak, is that right? Have you verified it works and communicates settings in real-time with the gnome extension?

Reply:
Also I don't see any explicit dependencies on Python's Gtk and Adw dependencies. This may work fine on a native GNOME install but I'm wondering if it works well in other distros for folks that are doing nested GNOME.

Reply:
What do you mean with communication in real time? If I enable it in the ui then the gnome-extension in the system tool also gets enabled. I think that counts as `connection` between ui and extension?

Reply:
As far as I can see, it all comes inside the `gnome-shell` package which is a hard requirement on [line 11](https://github.com/hodasemi/breezy-desktop/blob/2dc0053afaab8eebfdbdf7e900f9e625eff1bb71/PKGBUILD#L11).

Reply:
The extension and the UI both bind to the gsettings. If one writes, the other reflects the change in real-time (and vice-versa). The test for this would be:
1. Sliding the "display distance" slider in the UI is reflected visually in the display size as you slide it (no delay)
2. Using the display distance shortcut (assigned from the UI) is picked up and handled by the extension (not the UI), and it does an "easing" effect to slide the distance between two predefined distances over a short period of time. As the extension does this, the slider in the UI slides along at the same time, even though the extension is managing the easing.

Reply:
Yea, I worked with it over the weekend and everything works.